server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckInBadArgException.java

```diff
@@ -1,11 +1,8 @@
 package com.objectcomputing.checkins.services.checkins;
 
 public class CheckInBadArgException extends RuntimeException{
-     /**
-     *
-     */
-    private static final long serialVersionUID = 1L;
 
+    private static final long serialVersionUID = 1L;
```

Comment:
I don't believe in this. You have offended my religion and we must now duel to the death. If you're not willing to face me, you may substitute a champion or generate a proper serialVersionUID.

Reply:
pardon, my lord - for I am just a humble developer cleaning unused comments. I choose to clean the code over facing your wrath ;)